[2022-12-08T22:45:08.373+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: back_in_avro.First_task_backup manual__2022-12-08T22:45:06.807271+00:00 [queued]>
[2022-12-08T22:45:08.395+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: back_in_avro.First_task_backup manual__2022-12-08T22:45:06.807271+00:00 [queued]>
[2022-12-08T22:45:08.397+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-08T22:45:08.400+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 6
[2022-12-08T22:45:08.401+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-08T22:45:08.437+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): First_task_backup> on 2022-12-08 22:45:06.807271+00:00
[2022-12-08T22:45:08.448+0000] {standard_task_runner.py:55} INFO - Started process 7177 to run task
[2022-12-08T22:45:08.455+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'back_in_avro', 'First_task_backup', 'manual__2022-12-08T22:45:06.807271+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/dag_back_in_avro.py', '--cfg-path', '/tmp/tmpei3hg2dh']
[2022-12-08T22:45:08.458+0000] {standard_task_runner.py:83} INFO - Job 30: Subtask First_task_backup
[2022-12-08T22:45:08.552+0000] {task_command.py:389} INFO - Running <TaskInstance: back_in_avro.First_task_backup manual__2022-12-08T22:45:06.807271+00:00 [running]> on host c4fdd5f7672d
[2022-12-08T22:45:08.653+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Martin_Jurado
AIRFLOW_CTX_DAG_ID=back_in_avro
AIRFLOW_CTX_TASK_ID=First_task_backup
AIRFLOW_CTX_EXECUTION_DATE=2022-12-08T22:45:06.807271+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-08T22:45:06.807271+00:00
[2022-12-08T22:45:08.704+0000] {logging_mixin.py:137} INFO - There is information in the database
[2022-12-08T22:45:08.749+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-12-08T22:45:08.762+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=back_in_avro, task_id=First_task_backup, execution_date=20221208T224506, start_date=20221208T224508, end_date=20221208T224508
[2022-12-08T22:45:08.831+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-08T22:45:08.860+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
